src/lib.rs

```diff
@@ -103,6 +103,10 @@ fn build_poi_properties(
     Ok(properties)
 }
 
+fn iter_admins(admins: &[Arc<mimir::Admin>]) -> impl Iterator<Item = &mimir::Admin> + Clone {
+    admins.iter().map(|a| a.deref())
```

Comment:
`*a` instead of `a.deref()` isn't working?

Reply:
nope :(
```expected struct `std::sync::Arc`, found reference```

Reply:
Strange...